### DISCUSSION TITLE
deprecation warning

### DISCUSSION THREAD

**Discussion by Ta-Ara-Pro** [refinedev/refine#5906] [Q&A]
I am creating a react refine app and I am using following packages:

  "dependencies": {
    "@refinedev/cli": "^2.16.21",
    "@refinedev/core": "^4.47.1",
    "@refinedev/devtools": "^1.1.32",
    "@refinedev/kbar": "^1.3.6",
    "react": "^18.0.0",
    "react-dom": "^18.0.0",
    "react-router-dom": "^6.8.1",
    "@refinedev/nestjs-query": "^1.1.1",
    "graphql-tag": "^2.12.6",
    "graphql-ws": "^5.9.1",
    "@refinedev/antd": "^5.37.4",
    "antd": "^5.0.5",
    "@ant-design/icons": "^5.0.1",
    "@uiw/react-md-editor": "^3.19.5",
    "@refinedev/react-router-v6": "^4.5.5"
  },
  "devDependencies": {
    "@types/node": "^18.16.2",
    "@types/react": "^18.0.0",
    "@types/react-dom": "^18.0.0",
    "@typescript-eslint/eslint-plugin": "^5.57.1",
    "@typescript-eslint/parser": "^5.57.1",
    "@vitejs/plugin-react": "^4.0.0",
    "eslint": "^8.38.0",
    "eslint-plugin-react-hooks": "^4.6.0",
    "eslint-plugin-react-refresh": "^0.3.4",
    "typescript": "^4.7.4",
    "vite": "^4.3.1",
    "@graphql-codegen/cli": "^5.0.0",
    "@graphql-codegen/typescript": "^4.0.1",
    "@graphql-codegen/typescript-operations": "^4.0.1",
    "@graphql

when I try to sign-in in the sign-in page I come up with this error:
Cannot read properties of undefined (reading 'headers')
TypeError

and when I open developer tools I get this warnings:
warning.js:30  Warning: [antd: Card] `headStyle` is deprecated. Please use `styles.header` instead.
warning.js:30  Warning: [antd: Card] `bodyStyle` is deprecated. Please use `styles.body` instead.

I dont know how to fix it. Please help me out

**Comment by BatuhanW** [comment]
Hey @Ta-Ara-Pro which example are you trying to create?

**Comment by RAFA3L** [comment]
Hello, the problem is in this [line](https://github.com/refinedev/refine/blob/124618dd3f3ef04b61aa59633fd5ec7d6554feb1/packages/antd/src/components/pages/login/index.tsx#L67)

<Card title={CardTitle} headStyle={{ borderBottom: 0 }}>

Must be something like this

<Card title={CardTitle} styles={{ header: { borderBottom: 0 } }}>
